README.md

```diff
@@ -20,6 +20,45 @@ $ make VERSION=11.0  # generate image camptocamp/odoo-project:11.0-latest
 $ make VERSION=10.0  # generate image camptocamp/odoo-project:10.0-latest
 $ make VERSION=9.0  # generate image camptocamp/odoo-project:9.0-latest
 ```
+## Image version
+
+There is 4 versions of image:
+- odoo-project:${odoo_version}-${tag_version}
+- odoo-project:${odoo_version}-${tag_version}-onbuild
+- odoo-project:${odoo_version}-${tag_version}-full
+- odoo-project:${odoo_version}-${tag_version}-full-onbuild
+
+Note: in production, docker hardly recommanded to use never use latest tag but instead use a specific version in order to be able to rebuild identicall images.
```

Comment:
fix typos and repetitions: "Note: in production, docker docs strongly recommend to never use "latest" tags. Instead use a specific version in order to be able to rebuild identical images."